### PR TITLE
Add quick start for dapp development

### PR DESCRIPTION
**Pull request by polarker**

### CODE REVIEW

File: docs/dapps/Quick-Start.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+- Write code in [Typescript](https://www.typescriptlang.org/docs/handbook/typescript-in-5-minutes.html)
+- Operate a [terminal](https://en.wikipedia.org/wiki/Terminal_emulator)
+
+### Creating a new dApp project
+
+To create the tutorial project, open a new terminal and run:
+
+```
+npx @alephium/cli@latest init -d alephium-tutorial
```
Comment (Lbqds) [inline docs/dapps/Quick-Start.md]:
```suggestion
npx @alephium/cli@latest init alephium-tutorial
```

File: docs/dapps/Quick-Start.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+This will deploy the token faucet to all of the 4 groups of Alephium blockchain.
+
+You could read the comments of `scripts/0_deploy_faucet` to learn more about deployment.
+
+### Interacting with the deployed contract
+
+Now, you could build the source code `src/token.ts`:
+
+```
+npm run build
```
Comment (Lbqds) [inline docs/dapps/Quick-Start.md]: Do we need to add `npm install` before the `npm run build`?

**Review by Lbqds — APPROVED** [review]
LGTM 👍

File: docs/dapps/Quick-Start.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+---
+
+## Quick Start
+
+This guide will explore the basics of creating an Alephium dApp project.
+
+Prerequisites:
+
+- Write code in [Typescript](https://www.typescriptlang.org/docs/handbook/typescript-in-5-minutes.html)
+- Operate a [terminal](https://en.wikipedia.org/wiki/Terminal_emulator)
```
Comment (h0ngcha0) [inline docs/dapps/Quick-Start.md]:
```suggestion
- Operate in a [terminal](https://en.wikipedia.org/wiki/Terminal_emulator)
```

**Review by h0ngcha0 — COMMENTED** [review]
LGTM :+1:

**Pull request merged**